core/domain/exp_domain.py

```diff
@@ -3519,3 +3519,197 @@ def to_metadata_dict(self):
             'title': self.title,
             'objective': self.objective,
         }
+
+
+class StateIDMapping(object):
```

Comment:
We treat ID as one word. Can we rename this to StateIdMapping? This follows our existing conventions for names with things like e.g. "Html" in them.

Reply:
In the last commit, I renamed it from StateIDMapping to StateIdMapping. My bad. Done